src/lib/broker.ts

```diff
@@ -0,0 +1,30 @@
+export interface CoreViewer {
+  readyState: string;
+  addListener(type: string, listener: (payload: Payload) => void): void;
+  removeListener(type: string, listener: (payload: Payload) => void): void;
+  getMetadata?: () => Meta;
+  showTOC(opt_show?: boolean | null): void;
+  getTOC?: () => TOCItem[];
```

Comment:
We know the CoreViewer always has these methods in next release (https://github.com/vivliostyle/vivliostyle.js/pull/634) by specifying a version of `@vivliostyle/core`, so it seems `?` options are unnecessary👍  

```suggestion
  getMetadata: () => Meta;
  showTOC(opt_show?: boolean | null): void;
  getTOC: () => TOCItem[];
```

Reply:
Done. Tests will fail on this repo until core 2.1.0 is available.